src/appmixer/ai/openai/AIAgent/AIAgent.js

```
@@ -166,67 +300,73 @@
+            client,
+            context.properties.instructions || 'You\'re a helpful assistant.',
+            model,
+            prompt,
+            tools,
+            history
+        );
+        await context.log({ step: 'agent-response', response });
+
+        let newHistory = response.messages;
+        const newHistoryText = JSON.stringify(newHistory);
+        if (newHistoryText.length > context.config.AI_AGENT_MAX_HISTORY_SIZE || 10000) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)